packages/devextreme/js/__internal/ui/chat/chat_message_list.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import $ from '@js/core/renderer';
+import type { Message } from '@js/ui/chat';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+import MessageGroup from './chat_message_group';
+
+const CHAT_MESSAGE_LIST_CLASS = 'dx-chat-message-list';
+const CHAT_MESSAGE_LIST_CONTENT_CLASS = 'dx-chat-message-list-content';
+
+export interface MessageListOptions extends WidgetOptions<MessageList> {
+  items?: Message[];
+  currentUserId?: string;
+}
+
+class MessageList extends Widget<MessageListOptions> {
+  _messageGroup?: MessageGroup;
+
+  _getDefaultOptions(): MessageListOptions {
+    return {
+      ...super._getDefaultOptions(),
+      ...{
+        items: [],
+        currentUserId: '',
+      },
+    };
+  }
+
+  _initMarkup(): void {
+    $(this.element()).addClass(CHAT_MESSAGE_LIST_CLASS);
+
+    super._initMarkup();
+
+    this._renderMessageListContent();
+  }
+
+  _isCurrentUser(id): boolean {
+    const { currentUserId } = this.option();
+
+    return currentUserId === id;
+  }
+
+  _messageGroupAlignment(id): 'start' | 'end' {
+    return this._isCurrentUser(id) ? 'end' : 'start';
+  }
+
+  _renderMessageListContent(): void {
+    const $content = $('<ul>').addClass(CHAT_MESSAGE_LIST_CONTENT_CLASS);
```

Comment:
[markup]
It's really strange for me to render chat content as a List, at least because there will also be such element as errors and day headers. 
I expect there should be div element.
@EugeniyKiyashko  what do you think?
Maybe also designer can advise here

Reply:
I have an argument for and an argument against

Behind:

The list items are related in that they are all chat elements, yes, they have a different purpose: show date, display message group, display error. What unites them is that they all go in order in a common container. And it's like if we added navigation, users would move alternately from one element to the next

Against: 

Different purpose. It's true that elements carry different information

Reply:
I replaced it with a div